trojsten/notifications/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from trojsten.people.models import User
+
+
+class UnsubscribedChannel(models.Model):
```

Comment:
Optional: Rozmyslam ci by nebolo lepsie mat Subscription model ktory vie byt explicitne aj subscribed aj unsubscribed.
Ale ak vsetko co potrebujeme momentalne je default subscribed, tak to zatial mozme nechat takto a zmenit ked to bude treba.